docs/OpenSSL300Design.md

```diff
@@ -265,15 +273,25 @@ The physical packages new to this release are:
     there may be gaps/jumps in its version number since the previous
     release.
 
+*   ~~ Legacy module. This contains implementations of legacy
+    algorithms. ~~ This was not implemeted.
+
 *   Legacy module. This contains implementations of legacy
     algorithms.
 
-Engines will be built with a provider shim, to allow them to work as
+~~Engines will be built with a provider shim, to allow them to work as
 usual for cases when an ENGINE pointer is passed to some functions,
 and as providers when acting as default implementations. Engines
 compiled for pre-3.0.0 OpenSSL will need to be recompiled to work with
 OpenSSL 3.0.0. Details in
-[The ENGINE API](#the-engine-api) further down.
+[The ENGINE API](#the-engine-api) further down.~~
```

Comment:
Perhaps preface this with "It was originally intended that Engines would have a provider shim..."